packages/core/src/transforms/__testfixtures__/pureClassToFnComponent.input.js

```diff
@@ -1,6 +1,17 @@
 import React, { Component } from "react"
 
 class SimpleComponents extends Component {
+
```

Comment:
You should remove these as this file is only for the simple test case . The assumption is that all our transformation cases has to pass the initial checks or they fail.

Reply:
I think we should have a separate place to test out the initial condition. What do you think? This will require create a new/separate test runner function as the default setup assumes passing test means input and output are equal

Reply:
I agree. If `index.input.js` is just meant to check if input and output are the same, then that should be its only job.